tests/testUtils.js

```diff
@@ -84,5 +96,10 @@ module.exports = {
     .then(() => done())
     .catch((err) => done(err));
   }, // forceDeleteUser
-
+  // delete users
+  forceDeleteSubject(done) {
+    forceDelete(db.Subject, testStartTime)
+    .then(() => done())
+    .catch((err) => done(err));
+  }, // forceDeleteUser
 }; // exports
```

Comment:
If we use the same subject, its name will be changed in some tests. Hence it is a good idea to create and delete a separate subject for each test.